### PR TITLE
adds new toggle for presenter view

### LINKED ISSUE #295: add presenter view toggle button to current shabad (single monitor only)
maybe  to title or bottom right. This is useful in single monitor if you want to do search in presenter and then quickly switch to full slide.

perhaps @saintsoldierx can provide design and icon location guidance

### PR DESCRIPTION
**Pull request by inderpreetsingh**
ezgif-3-981faf951bcb

It fixes #295 

I have tested it with chromecast and single monitor. Yet to test it with a dual monitor setup.

### CODE REVIEW

**Comment by navdeepsinghkhalsa** [comment]
@saintsoldierx Could you please sign off on design before code review?

**Comment by inderpreetsingh** [comment]
@navdeepsinghkhalsa took a verbal sign off from aman veerji over call! 🙈 @saintsoldierx

File: www/index.html [hunk 1]
```diff
@@ -167,12 +167,13 @@
       <div id="footer">
         <div class="menu-group">
           <div class="menu-group-left"></div>
+          <div class="presenter-switch"> </div>
```
Comment (tsingh777) [inline www/index.html]: should we remove the one in the settings eventually? doesn't have to be in this pr.

File: www/js/controller.js [hunk 2]
```diff
@@ -386,7 +386,12 @@ function checkPresenterView() {
     document.body.classList.add('presenter-view');
     document.body.classList.remove('home');
     document.body.classList.add('scale-viewer');
+  } else {
+    document.body.classList.remove('presenter-view');
+    document.body.classList.add('home');
+    document.body.classList.remove('scale-viewer');
   }
+  document.querySelector('#presenter-view-toggle').checked = store.getUserPref('app.layout.presenter-view');
```
Comment (tsingh777) [inline www/js/controller.js]: could refactor this function to less lines with something like
```
function checkPresenterView() {
  const inPresenterView = store.getUserPref('app.layout.presenter-view');
  const classList = document.body.classList;

  classList.toggle('presenter-view', inPresenterView);
  classList.toggle('home', !inPresenterView);
  classList.toggle('scale-viewer', inPresenterView);

  document.querySelector('#presenter-view-toggle').checked = inPresenterView;
}
```

your call.

File: www/index.html [hunk 3]
```diff
@@ -167,12 +167,13 @@
...
           <!-- <div class="menu-group-right">
-            <ul class="menu-bar">
+             <ul class="menu-bar">
```
Comment (tsingh777) [inline www/index.html]: commented code should be removed eventually.

File: www/index.html (hunk 1 above)
Comment (inderpreetsingh) [reply www/index.html]: Yeah need to discuss that with aman veerji. All these options are now at two places (specially display options).

File: www/index.html (hunk 3 above)
Comment (inderpreetsingh) [reply www/index.html]: Yeah, I agree. I will remove it.

**Comment by maneetpaul** [comment]
Tracking this in #295 - are we all set with this PR?

**Comment by inderpreetsingh** [comment]
Yeah, its ready from my side. @maneetpaul

**Pull request merged**